### PR TITLE
New arab latin-script file  

### PR DESCRIPTION
**Pull request by AhmedRedaAmin**
The most recent version of the list compiled for get-alex/alex#220

### CODE REVIEW

File: index-ar.json [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  "zega": 2,
+  "zegg": 2,
+  "zegga": 2,
+  "zippak": 2,
+  "zippy": 2,
+  "zobr": 2,
+  "zobrak": 2,
+  "zobry": 2,
+  "zwamel": 2
+}
```
Comment (wooorm) [inline index-ar.json]: Ohh, could you add a newline at the end of the file?

**Comment by AhmedRedaAmin** [comment]
Okay @wooorm  done !

**Comment by wooorm** [comment]
OK, two more things.
Could you add the inspiration for this file (where you got the phrases from), and an entry in the [language support section](https://github.com/words/cuss#language-support)?

**Comment by AhmedRedaAmin** [comment]
@wooorm , I already have the sources on my repo , how do you want me to transfer them ?

**Comment by wooorm** [comment]
@AhmedRedaAmin You need to add them to this branch. If you push them to the master branch on your fork (GitHub says) they’re added here automatically!

**Comment by AhmedRedaAmin** [comment]
Lol @wooorm  , I know how to make a pull request Titus , I was more asking about the format that you expect from me , a list in a separate file , embedding them as a comment in the .json , perhaps adding them to the readme ? You know , to keep the project structure nice and uniform .

**Comment by AhmedRedaAmin** [comment]
Okay , this was more difficult than anticipated , @wooorm any hint on the rules at work here ?

**Comment by wooorm** [comment]
@AhmedRedaAmin Landed!!

**Pull request merged**